src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## Unreleased
 
+* Added support for Activity Status and StatusDescription which were
```

Comment:
Please modify the changelog for other exporters as well, to steal from this. looks very nice.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/3104